### DISCUSSION TITLE
Copy ffmpeg command/flags button

### DISCUSSION THREAD

**Discussion by BuyMyMojo** [Xaymar/obs-stream-effects#850] [Ideas & Feedback]
### Explain the Feature/Enhancement

For testing purpouses a button that would copy the ffmpeg flags generated by the options chosen from StreamFX's different encoders would be nice.

### What makes this feature/enhancement useful for StreamFX and its users?

Comparing the quality of different options easily with a consistent video input

### Checklist

- [ ] I am offering a monetary bounty/reward for the implementation of this Feature/Enhancement.
- [X] There is no other way to currently do this in StreamFX or OBS Studio.

### Any additional Information we need to know?

I personally think this addition is verry low priority but it'd still be nice to have

**Comment by Xaymar** [comment]
This will most likely not work out. A number of settings are decided during encoder initialization, and thus can't be exported without fully initializing the encoder. I could potentially add the generated settings to the log file though, it should hopefully be possible.
